### PR TITLE
Add Confluence Persistence

### PR DESCRIPTION
**Pull request by david--wright**
Implements #91 Adds a persistent volume claim to allow for pod restarts without reconfiguration. Using local-path for demo storage class.

### CODE REVIEW

**Comment by jeff-mccoy** [comment]
Is 1 gb a reasonable size?

**Comment by david--wright** [comment]
> Is 1 gb a reasonable size?
This is the default from the upstream values file. So far as I can tell these folders hold the configuration files. All actual user data is stored in the data base. We have not seen issues with this setting so far.

**Comment by jeff-mccoy** [comment]
Since we can't spin up tests right now during the retreat until late evening, if you'd be able to drop a screenshot of it working on the PRs I'll approve now. If not, I'll be able to look at late tonight or tomorrow.

**Comment by david--wright** [comment]
I don't have an instance up with just this code. I tested it in our customized repo so it has other configuration changes. I think it would be best to just wait. Assuming other changes aren't going to be made to this file it isn't urgent to get this in. It could even wait until you are back from the retreat.

**Comment by jeff-mccoy** [comment]
Okay sounds great thanks!

File: kustomizations/softwarefactoryaddons/confluence/common-values.yaml [hunk 1]
```diff
@@ -34,3 +34,24 @@ confluence:
         secretKeyRef:
           name: "confluence.acid-confluence.credentials.postgresql.acid.zalan.do"
           key: "password"
+  volumes:
+    localHome:
+      persistentVolumeClaim:
+        # -- If true, then a PersistentVolumeClaim will be created for each local-home volume.
+        create: true
+        # -- Specifies the name of the storage class that should be used for the local-home volume claim.
+        storageClassName: local-path
```
[the remark this answers is not in the document]
Comment (jeff-mccoy) [reply kustomizations/softwarefactoryaddons/confluence/common-values.yaml:43]: Yep that will break anything but k3s (even kind which uses the same storage class actually uses a different name).

**Comment by RothAndrew** [comment]
I'm gonna pull this one and the Jira one into one branch, test both, make the couple of changes that were mentioned, and get it merged. Thanks for the contribution @david--wright!

**Pull request merged**